app/actions/transactions.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_TRANSACTION = 'ADD_TRANSACTION'
```

Comment:
Question, are those actions directly related to a specific "view" or "route" or are actions used across all the app?

Reply:
They are used across all the app.